documentation/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "docusaurus": "docusaurus",
-    "start": "docusaurus start --host 0.0.0.0",
+    "start": "docusaurus start --host 0.0.0.0 --port 3030",
```

Comment:
Hey @huynguyen-hl, just asking, is there an alternative to hardcoding 3030?

Reply:
@ldhyen99, I don't have another solution to change the port here. Do you have one?

Reply:
Oh, I am too...